enhancements/monitoring/user-workload-monitoring.md

```diff
@@ -291,6 +313,90 @@ remapping the values of labels as we base our tenancy model on this. We have two
 prometheus-operator, either discard that object completely in the same way as we do with filesystem access right now or
 create the respective objects and change the honorLabel value to false.
 
+### Roles
+
+The following new named roles are deployed via cluster-monitoring-operator.
+
+#### `monitoring-rules-view`
+This is a cluster role which can be bound against a concrete namespace by the cluster admin.
```

Comment:
We need newline here for proper markdown rendnering. Ditto below.

Reply:
ack :+1: